### PR TITLE
Update bazel

### PR DESCRIPTION
**Pull request by alexeagle**

### CODE REVIEW

**Comment by filipesilva** [comment]
@alan-agius4 this should unblock parts of https://github.com/angular/angular/issues/29785

File: packages/bazel/src/schematics/ng-add/index.ts [hunk 1]
```diff
@@ -48,7 +48,7 @@ function addDevDependenciesToPackageJson(options: Schema) {
 
     const devDependencies: {[k: string]: string} = {
       '@angular/bazel': angularCoreVersion,
-      '@bazel/bazel': '^0.25.1',
+      '@bazel/bazel': '^0.26.0-rc.5',
       '@bazel/ibazel': '^0.10.2',
       '@bazel/karma': '0.27.12',
```
Comment (kyliau) [inline packages/bazel/src/schematics/ng-add/index.ts]: `@bazel/karma` and `@bazel/typescript` should be upgraded to `0.29.0` as well.

**Comment by alexeagle** [comment]
I'm on vacation next week so I could use some help to get this merged.

File: packages/core/test/linker/integration_spec.ts [hunk 2]
```diff
@@ -1567,7 +1567,7 @@ function declareTests(config?: {useJit: boolean}) {
                 expect(getDOM().nodeName(c.renderNode).toUpperCase()).toEqual('SPAN');
                 expect(getDOM().nodeName(c.componentRenderElement).toUpperCase()).toEqual('DIV');
                 expect((<Injector>c.injector).get).toBeTruthy();
-                expect(c.context).toBe(fixture.componentInstance);
+                expect(c.context).toEqual(fixture.componentInstance);
```
Comment (alan-agius4) [inline packages/core/test/linker/integration_spec.ts]: Is this expected to be the same instance or just to be deep equal?
`toBe`  means it has to be the same instance ie it's checked with `===`

**Comment by kyliau** [comment]
Investigating the failure on CodeFresh ...

**Comment by alan-agius4** [comment]
@kyliau, I am seeing failures on other builds in CodeFresh all with OOM errors.

**Comment by filipesilva** [comment]
We're following up with the Codefresh folks. It's not related to this PR.

**Comment by kyliau** [comment]
Caretaker, per Alan and Filipe's comments, the failure on CodeFresh is unrelated to this PR. Please merge, thank you!

**Review by alan-agius4 — APPROVED** [review]

**Pull request closed without merging**